src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
> there's nothing stopping other plugins from potentially adopting it in the future.

I agree, but there currently, **as today**, no use case to do it now. Instead, when we will have other plugin that will repeat the same piece of code, only then centralizing into an extend will be the best solution to take.